<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lotes</class>
 <widget class="QDialog" name="lotes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Factura predef.:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="predefinidascomboBox">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="descrip_predeflabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Descripción</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Añadir cuentas auxiliares de código principal:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="principallineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="addctaspushButton">
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../keme.qrc">
         <normaloff>:/iconos/graf/reload2.png</normaloff>:/iconos/graf/reload2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cuotascheckBox">
       <property name="text">
        <string>En base a cuotas de participación</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>Cuenta</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Descripción</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Cuota</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nuevo importe</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Externo</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="edpushButton">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../keme.qrc">
           <normaloff>:/iconos/graf/edit.png</normaloff>:/iconos/graf/edit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addpushButton">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../keme.qrc">
           <normaloff>:/iconos/graf/Files-New-File-icon.png</normaloff>:/iconos/graf/Files-New-File-icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="suppushButton">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../keme.qrc">
           <normaloff>:/iconos/graf/borrar.png</normaloff>:/iconos/graf/borrar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="presupuesto_label">
       <property name="text">
        <string>Presupuesto:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="presupuestolineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="cuotas_label">
       <property name="text">
        <string>Suma cuotas:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sumacuotaslineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Fecha documentos:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="fecha_dateEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelarpushButton">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="../keme.qrc">
         <normaloff>:/iconos/graf/borraelem.png</normaloff>:/iconos/graf/borraelem.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generarpushButton">
       <property name="text">
        <string>&amp;Generar docs.</string>
       </property>
       <property name="icon">
        <iconset resource="../keme.qrc">
         <normaloff>:/iconos/graf/run-build.png</normaloff>:/iconos/graf/run-build.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptarpushButton">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="../keme.qrc">
         <normaloff>:/iconos/graf/application-exit.png</normaloff>:/iconos/graf/application-exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>lotes</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
